cfme/tests/containers/test_chargeback.py

```diff
@@ -125,23 +125,23 @@ def gen_report_base(appliance, obj_type, provider, rate_desc, rate_interval):
     return report
 
 
-def assign_custom_compute_rate(obj_type, chargeback_rate, provider):
+def assign_custom_compute_rate(obj_type, chargeback_rate, provider, appliance):
     """Assign custom Compute rate for Labeled Container Images
     Args:
         :py:type:`str` obj_type: Object being tested; only 'Project' and 'Image' are supported
         :py:class:`ComputeRate` chargeback_rate: The chargeback rate object
         :py:class:`ContainersProvider` provider: The containers provider
     """
     if obj_type == 'Image':
-        asignment = assignments.Assign(
-            assign_to="Labeled Container Images",
-            docker_labels="architecture",
-            selections={
+        asignment = appliance.collections.assignments.Assign(
+            "Labeled Container Images",
+            "architecture",
+            {
                 'x86_64': {'Rate': chargeback_rate.description}
             })
         logger.info('ASSIGNING COMPUTE RATE FOR LABELED CONTAINER IMAGES')
     elif obj_type == 'Project':
-        asignment = assignments.Assign(
+        asignment = appliance.collections.assignments.Assign(
```

Comment:
REQUIRED: why do you call Assign directly here instead of calling instantiate method ?